core/src/components/picker-column/picker-column.tsx

```diff
@@ -465,22 +478,161 @@ export class PickerColumn implements ComponentInterface {
     this.el.classList.remove('picker-column-active');
   };
 
-  get activeItem() {
+  get activeItem(): {
+    activeItem?: HTMLIonPickerColumnOptionElement;
+    activeItemIndex: number;
+  } {
     const { value } = this;
-    const options = Array.from(this.el.querySelectorAll<HTMLIonPickerColumnOptionElement>('ion-picker-column-option'));
-    return options.find((option) => {
+    const options = this.el.querySelectorAll<HTMLIonPickerColumnOptionElement>('ion-picker-column-option');
+
+    for (let i = 0; i <= options.length - 1; i++) {
+      const option = options[i];
+
       /**
        * If the whole picker column is disabled, the current value should appear active
        * If the current value item is specifically disabled, it should not appear active
        */
-      if (!this.disabled && option.disabled) {
-        return false;
```

Comment:
My understanding is this if statement had more checks than it needed to. According to the comment, the picker column's disabled state has no bearing on whether or not an option should be rendered as active, so we don't need to check it.